packages/sacado/cmake/Sacado_config.h.in

```diff
@@ -29,10 +29,10 @@
 #cmakedefine HAVE_SACADO_TEUCHOSNUMERICS
 
 /* Define if want to build with kokkos-core enabled */
-#cmakedefine HAVE_SACADO_KOKKOSCORE
+#cmakedefine HAVE_SACADO_KOKKOS
 
 /* Define if want to build with kokkos-containers enabled */
-#cmakedefine HAVE_SACADO_KOKKOSCONTAINERS
+#cmakedefine HAVE_SACADO_KOKKOS
```

Comment:
probably don't need this second define anymore

Reply:
@rppawlo, yes, that is the additional clean I was referring to in the commit message [`37a0259` (#11808)](https://github.com/trilinos/Trilinos/pull/11808/commits/37a025906abb990e0f36d669648e773b23dbd206) and https://github.com/trilinos/Trilinos/pull/11808/files#diff-834265fa841a516cc293104d8f66988af1600be94c7010e3e5adda2a36629afc.

It would be hard to write a script to robustly automatically remove this type of duplication but we can always to a later manual cleanup.

Do you think we should remove all of this duplication manually before merging this PR or can we do it in a later PR?  I was hopping to get this merged and have downstream customers like EMPIRE, Albany, Xyce, Sierra, SPARC, Nalu and others test against this to see if there are major problems in case we need to back this out.  (I added a task for a follow-up PR to to cleanup this duplication in #11545.)

Reply:
separate pr is fine for removing duplication.